src/main/java/seedu/address/logic/commands/DeleteGroupCommand.java

```diff
@@ -43,7 +43,10 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_GROUP_DOES_NOT_EXIST);
         }
 
-        model.deleteGroup(group);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, group));
+        Group groupFromInternalList = model.getGroupByName(group.getGroupName());
+        model.deleteGroup(groupFromInternalList);
+
+        return new CommandResult(String.format(MESSAGE_SUCCESS, groupFromInternalList.getGroupName(),
+                groupFromInternalList.getMembers().size()));
```

Comment:
@ramapriyan912001 Perhaps this may come off as a little confusing when other students are reviewing are code quality. Should we break this down into something that is easier to understand? (E.G String successMessage = ...)

Reply:
Good catch Chrus. 
Will fix it.